src/components/Header.tsx

```diff
@@ -20,26 +21,26 @@ export class Header extends React.Component<HeaderProperties> {
     leftButtonType: 'menu'
   };
 
-  static height = 50;
+  static height = isTablet() ? 65 : 50;
 
   getStyles = (): any => {
     const theme = getTheme();
 
     const styles = StyleSheet.create({
       header: {
         backgroundColor: theme.backgroundColor3,
-        height: Header.height || 40,
+        height: Header.height || isTablet() ? 60 : 40,
```

Comment:
```suggestion
        height: Header.height,
```

`Header.height` will always be defined, so you don't need the '||' part.